src/Blazor.Extensions.AzureMaps/Client/src/BE.AzureMaps.ts

```diff
@@ -108,4 +121,27 @@ export class BEAzureMaps {
     }
     this._map.events.add("ready", () => new dtools.drawing.DrawingManager(this._map, options));
   };
+
+  static drawTiles(coordinates: any) {
+    this._tilesDataSource = <atlas.source.DataSource>(this._map.sources.getById("tilesDataSource"));
+    if (!this._tilesDataSource) {
+      this._tilesDataSource = new atlas.source.DataSource("tilesDataSource");
+      this._map.sources.add(this._tilesDataSource);
+    }
+    
+    //Create a polygon and add it to the data source.
+    this._tilesDataSource.add(new atlas.Shape(
+      new atlas.data.Feature(
+        new atlas.data.Polygon([coordinates])
+    )));
+
+    //Create a polygon layer to render the filled in area of the polygon.
+    this._polygonLayer = new atlas.layer.PolygonLayer(this._tilesDataSource, "myPolygonLayer", {
+      fillColor: "rgba(255,165,0,0.2)",
```

Comment:
Why are those fields hardcoded? Shouldn't the user be able to send the parameters with the follow, etc when they are creating the tiles?